webapp/packages/plugin-connections/src/ConnectionForm/Options/Options.tsx

```diff
@@ -149,7 +149,7 @@ export const Options: TabContainerPanelComponent<IConnectionFormProps> = observe
   });
 
   const edit = state.mode === 'edit';
-  const originLocal = !info || (originInfo && isLocalConnection(originInfo));
+  const originLocal = !info || (originInfo?.origin && isLocalConnection(originInfo.origin));
```

Comment:
isLocalConnection can accepts origin | undefined (in case property origin can be undefined / null)

Reply:
After the new changes now it is not able to pass `null` or `undefined`